app/components/base/table/table.scss

```diff
@@ -31,7 +31,7 @@ bl-table {
         display: table-header-group;
 
         > tr {
-            cursor:pointer;
```

Comment:
ill push it like this as i think its ok and in-line with tables in the portal. though if we decide otherwise we can add a specific style to this component.